calendar/v0.1.0/schema.json

```diff
@@ -0,0 +1,79 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://common.schemas.example.com/calendar/v0.1.0/schema.json",
+  "title": "Calendar",
+  "titlePlural": "Calendars",
+  "description": "Google Calendar information",
+  "type": "object",
+  "appearance": {
+    "style": {
+      "color": "#42A5F5",
+      "icon": "./calendar-icon.svg"
+    }
+  },
+  "database": {
+    "name": "calendar_data",
+    "indexes": {
+      "id": [
```

Comment:
The id field is always called `_id`. It is always indexed by default as well, so you can remove this index.